src/python/pants/backend/python/subsystems/python_setup.py

```diff
@@ -62,13 +62,19 @@ def register_options(cls, register):
     register('--artifact-cache-dir', advanced=True, default=None, metavar='<dir>',
              help='The parent directory for the python artifact cache. '
                   'If unspecified, a standard path under the workdir is used.')
+    register('--interpreter-search-paths', advanced=True, type=list, default=[], metavar='<binary-paths>',
+             help='A list of paths in which to search for python')
```

Comment:
"To search for python interpreters."

Tiny nit: End the help sentence with a period. For grammar and for consistency...

Reply:
@benjyw fixed 😄